Client/Application/AppDelegate+Experiments.swift

```diff
@@ -10,17 +10,16 @@ import Foundation
 
 extension AppDelegate {
     func initializeExperiments() {
-
         let defaults = UserDefaults.standard
         let nimbusFirstRun = "NimbusFirstRun"
-        let firstRun = defaults.bool(forKey: nimbusFirstRun) != false
+        let isFirstRun = defaults.object(forKey: nimbusFirstRun) == nil
```

Comment:
Looks like the `firstRun` indicator here didn't work correctly, so I changed it to `object` to do the `nil` check, which would only be true on the first run

 (`defaults.bool` would have always returned false)

Reply:
Doh, good catch. I forgot that `bool(forKey:)` doesn't return null like almost _all_ of the other functions in that API.

Reply:
I had to check for the existence of the key for this same reason in Kotlin.